renovate.json

```diff
@@ -32,7 +28,22 @@
     },
     {
       "matchPackageNames": ["go.opentelemetry.io/otel/**"],
-      "allowedVersions": "/^v[0-9]+\\.[0-9]+\\.[0-9]+/"
+      "allowedVersions": "^v[0-9]+\\.[0-9]+\\.[0-9]+",
+      "enabled": true,
+      "ignore": {
+        "matchUpdateTypes": ["pin", "digest"]
+      }
+    },
+    {
+      "matchPackageNames": ["go.opentelemetry.io/otel/**"],
+      "sourceUrlPrefixes": ["https://github.com/open-telemetry/opentelemetry-go"],
+      "enabled": false,
+      
+      "_comment": {
```

Comment:
This `_comment` section is a neat trick.
However, I think it would be easier to read (especially because of folks habits) if it were actual yaml comments rather than a string which may seem like part of the configuration to an untrained eye.

Reply:
I've tried to do something about the comments , since its my first time adding yaml comments in a json file I'm not sure if i did it right open for corrections. TIA